eclair-core/src/test/scala/fr/acinq/eclair/router/RouterSpec.scala

```diff
@@ -1040,49 +1095,4 @@ class RouterSpec extends BaseRouterSpec {
     }
   }
 
-  test("properly announce valid new nodes announcements and ignore invalid ones") { fixture =>
```

Comment:
I moved this test (without changes) because it made sense to keep it closer to a very similar test for channels.